brain_games/scripts/brain_calc.py

```diff
@@ -0,0 +1,49 @@
+from ..cli import welcome_user
+import prompt
+from random import randint, choice
+
+
+name = welcome_user()
+print('What is the result of the expression?')
+
+
+def verification_expression(number_1, random_math_sign1, number_2):
```

Comment:
Лучше метод назвать глаголом, первый и второй параметры операнды, третий operation
```suggestion
def calculate(number_1, number_2, operation):
```